extras/dht/dht.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __DTH_H__
+#define __DHT_H__
+
+#include <espressif/esp_misc.h> // sdk_os_delay_us

```

Comment:
(nitpick) There's nothing in this header file that references `sdk_os_delay_us`, so there's no reason that `dht.h` should include `esp_misc.h`..  That should really go in `dht.c` instead.